README.md

```diff
@@ -90,7 +161,7 @@ Borges has 2 runtime dependencies and has tests that depend on them:
     Consumers make SIVA files with RootedRepositories, but all repository metadata is stored in PostgreSQL.
     You can run one in Docker with the following command:
     ```
-    docker run --name postgres -e POSTGRES_PASSWORD=testing -p 5432:5432 -e POSTGRES_USER=testing -d postgres
+    docker run --name postgres  -e POSTGRES_USER=testing -e POSTGRES_PASSWORD=testing  -p 5432:5432 -d postgres
```

Comment:
Doesn't the default configuration of the database in core use the database `testing`? If so, you're missing `-e POSTGRES_DB=testing`